### PR TITLE
LRQA-42766 | master

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LRQA-42766

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by weller0246** [comment]
ci:test:ldap:nocompile

**Comment by weller0246** [comment]
@gaborlovasdotcom the PR is  ready for your wisdom

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase [hunk 1]
```diff
@@ -354,4 +354,36 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-42766. Users can be imported with group import method."
+	@priority = "3"
+	test UsersCanBeImportedFromLDAPByGroups {
+		property apacheds.enabled = "true";
+		property portal.upstream = "marcell";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase:361]: @weller0246 Please remove this property

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Please remove CI only changes

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase (hunk 1 above)
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase]: Solved

**Comment by gaborlovasdotcom** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**